tests/api_tests/api_tests.cpp

```diff
@@ -211,9 +222,13 @@ BOOST_FIXTURE_TEST_CASE(test_all, testing_fixture)
       chain.produce_blocks(1);
 
       //Set test code
-      SetCode(chain, "test_api", test_api_wast_str.c_str());
-      SetCode(chain, "acc1", test_api_wast_str.c_str());
-      SetCode(chain, "acc2", test_api_wast_str.c_str());
+      types::setcode handler;
+      handler.code.resize(wasm.size());
+      memcpy( handler.code.data(), wasm.data(), wasm.size() );
```

Comment:
Why memcpy if you don't have to? `handler.code.assign(wasm.begin(), wasm.end());` does the resize and assign without worrying about buffer boundaries. :)